modules/zipkin-http-exporter/src/test/scala/io/janstenpickle/trace4cats/zipkin/ZipkinHttpSpanExporterSpec.scala

```diff
@@ -24,9 +26,12 @@ class ZipkinHttpSpanExporterSpec extends BaseJaegerSpec {
           )
         )
       )
+    val exporter = BlazeClientBuilder[IO](global).resource.flatMap { client =>
```

Comment:
This is a deprecated API. TODO: update trace4cats-jaeger-integration-test

Reply:
@catostrophe I saw the early release, done 🙂